pkg/runtime/typescript.dockerfile

```diff
@@ -14,25 +17,32 @@ COPY . .
 
 RUN yarn import || echo Lockfile already exists
 
-RUN set -ex; yarn install --production --frozen-lockfile --cache-folder /tmp/.cache; rm -rf /tmp/.cache;
+RUN set -ex; yarn install --frozen-lockfile --cache-folder /tmp/.cache; rm -rf /tmp/.cache;
 
 RUN test -f tsconfig.json || echo "{\"compilerOptions\":{\"esModuleInterop\":true,\"target\":\"es2015\",\"moduleResolution\":\"node\"}}" > tsconfig.json
 
-RUN ncc build ${HANDLER} -m --v8-cache -o lib/
+# make prisma external to bundle - https://github.com/prisma/prisma/issues/16901#issuecomment-1362940774 \
+# TODO: remove when custom dockerfile support is available
+RUN ncc build ${HANDLER} -o lib/ -e .prisma/client -e @prisma/client
 
 FROM node:alpine as final
 
+WORKDIR /usr/app
+
 RUN apk update && \
     apk add --no-cache ca-certificates && \
     update-ca-certificates
 
-COPY --from=build "package.json" "package.json"
+COPY --from=build /usr/app/lib/ ./lib/
 
-COPY --from=build "node_modules/" "node_modules/"
+COPY . .
 
-COPY --from=build lib/ /
+RUN set -ex; \
+    yarn install --production --frozen-lockfile --cache-folder /tmp/.cache; \
+    rm -rf /tmp/.cache; \
+    # prisma fix for docker installs: https://github.com/prisma/docs/issues/4365
+    # TODO: remove when custom dockerfile support is available
+    test -d ./prisma && npx prisma generate || echo "";
```

Comment:
It's worth noting that this line _could_ be removed from our container build and developers could add `npx prisma generate` to a postInstall hook. However, since we know prisma's popularity, it seem that would introduce unneeded friction for new users of nitric.